physionet-django/project/templates/project/published_project.html

```diff
@@ -312,8 +312,8 @@ <h5 class="card-header">Versions</h5>
     </div>
     <!-- /.sidebar -->
   </div>
-
   <h2 id="files">Files</h2>
+  {% if project.allow_file_downloads %}
```

Comment:
Minor thing but easier to navigate these nested `if` statements if they are pushed in with a couple of spaces.